dashboard/db/migrate/20190208001843_create_user_school_infos.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUserSchoolInfos < ActiveRecord::Migration[5.0]
+  def change
+    create_table :user_school_infos do |t|
+      t.integer  :user_id
```

Comment:
i think you'll want to add `null: false` to some of these columns, although i'm not too familiar with what this model should look like. we should validate that this user_school_info is associated with a user and a school (i.e., add `null: false` to `user_id` and `school_info_id`), but i'm not sure about the other fields (`start_date`, `end_date`, and `last_confirmation_date`) and whether they should be mandatory.

once you make these columns required, you should also add validations to the model like:
```
validates_presence_of :user
validates_presence_of :school_info
...
```

Reply:
I believe end_date can be null but the rest shouldn't be.